delivery_carrier_label_batch/tests/test_generate_labels.py

```diff
@@ -15,14 +16,14 @@ class TestGenerateLabels(common.SavepointCase):
     def setUpClass(cls):
         super(TestGenerateLabels, cls).setUpClass()
 
-        Move = cls.env["stock.move"]
-        Picking = cls.env["stock.picking"]
-        ShippingLabel = cls.env["shipping.label"]
-        BatchPicking = cls.env["stock.picking.batch"]
-        cls.DeliveryCarrierLabelGenerate = cls.env["delivery.carrier.label.generate"]
+        stock_move = cls.env["stock.move"]
+        stock_picking = cls.env["stock.picking"]
+        shipping_label = cls.env["shipping.label"]
+        batch_picking = cls.env["stock.picking.batch"]
+        cls.picking_batch_apply_carrier = cls.env["picking.batch.apply.carrier"]
+        cls.label_generate_wizard = cls.env["delivery.carrier.label.generate"]
```

Comment:
These changes are not mandatory and could lead to some conflicts when porting this fix on other branches, even if we don't like these names :)
Better to handle this during a migration.